src/ImageSharp.Drawing/Processing/Processors/Drawing/FillRegionProcessor.cs

```diff
@@ -139,27 +139,42 @@ protected override void OnFrameApply(ImageFrame<TPixel> source, Rectangle source
                                 {
                                     for (float x = scanStart; x < startX + 1; x += subpixelFraction)
                                     {
-                                        scanline[startX] += subpixelFractionPoint;
-                                        scanlineDirty = true;
-                                    }
+                                        scanline[startX] += subpixelFractionPoint;
+                                        scanlineDirty = true;
+                                    }
+
+                                    if (scanline[startX] > 1)

```

Comment:
What do these new constraining checks do?

Reply:
When I had the DebugGuard set, FillRegionProcessor was producing some opacities of value 1.02xxx... the algorythms there probably need to be reviewed to ensure that opacity values are always in range. But since I don't fully understand what these functions do, I just added a clamp there.

Now that the clamps are back into the pixels, these checks could be removed...